doc/man3/SSL_get_error.pod

```diff
@@ -181,9 +182,13 @@ connection and SSL_shutdown() must not be called.
 
 =back
 
+The OpenSSL error queue can be diagnosed with the B<ERR> family of functions,
+such as L<ERR_print_errors(3)> and L<ERR_peek_last_error_line_data(3)>.
+
 =head1 SEE ALSO
 
-L<ssl(7)>
+L<ssl(7)>,
+L<ERR_clear_error(3)>, ERR_print_errors(3), ERR_peek_last_error_line_data(3)
```

Comment:
`ERR_peek_last_error_line_data` is deprecated. Could you please update it to a new version of this function?
The same for all the occurrences in the PR